cmd/bisync/queue.go

```diff
@@ -358,3 +372,16 @@ func (b *bisyncRun) saveQueue(files bilib.Names, jobName string) error {
 	queueFile := fmt.Sprintf("%s.%s.que", b.basePath, jobName)
 	return files.Save(queueFile)
 }
+
+func naptime(totalWait time.Duration) {
```

Comment:
This is cute!

The implementation we use in cmd is here :-)

https://github.com/rclone/rclone/blob/91b54aafccc3aed01a5389864e8ef3902286b406/cmd/cmd.go#L293

That made me think though - could `bisync` use the `cmd.Run` function which does `--retries` and `--retries-sleep` for the other commands? That would make it more consistent. You could improve the function to make it work better with this nice function

https://github.com/rclone/rclone/blob/91b54aafccc3aed01a5389864e8ef3902286b406/cmd/cmd.go#L243

Reply:
This occurred to me too, and `bisync` does already use `cmd.Run`, but it deliberately does so with [`Retry = false` ](https://github.com/rclone/rclone/blob/91b54aafccc3aed01a5389864e8ef3902286b406/cmd/bisync/cmd.go#L195)-- the reason is because using the standard `Retry` here would retry the entire command (in both directions), which could cause data loss given the bidirectional nature of `bisync` (for example, an errored file in one direction could get synced back in the other direction on the next retry.) The way bisync gets around this is by retrying in only one direction at a time (and if none of the retries are successful it will abort rather than proceed in the other direction.) I suppose they are really more mid-level retries than high-level 🙂 

So I think this precludes bisync from using the standard `cmd.Run` retries, but it definitely could at least reference the global flag values if there's a way to do that! (See my other comment with questions about this.)